src/test/java/de/retest/web/it/RecheckRemoteWebElementIT.java

```diff
@@ -0,0 +1,47 @@
+package de.retest.web.it;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import de.retest.recheck.Recheck;
+import de.retest.recheck.RecheckImpl;
+import de.retest.web.selenium.By;
+import de.retest.web.testutils.PageFactory;
+import de.retest.web.testutils.PageFactory.Page;
+
+public class RecheckRemoteWebElementIT {
+
+	WebDriver driver;
+	Recheck re;
+
+	@BeforeEach
+	void setUp() {
+		re = new RecheckImpl();
+	}
+
+	@ParameterizedTest
+	@MethodSource( "de.retest.web.testutils.WebDriverFactory#drivers" )
+	void simple_html_page_should_be_checked( final WebDriver driver ) throws Exception {
+		this.driver = driver;
+		re.startTest( "simple-page-" + driver.getClass().getSimpleName() );
+
+		driver.get( PageFactory.page( Page.PAGE_FRAME ) );
+		Thread.sleep( 1000 );
+		final WebElement email = driver.findElement( By.id( "email" ) );
+
+		re.check( email, "open" );
```

Comment:
Things that come to mind:
1. What if I insert a html node, does this still work like a driver (without additional window information)
2. What if the node has (no) children
3. What if the node is inserted/deleted
4. Too many changes (e.g. wrong node) since the algorithm has quite the trouble with root changes (i.e html[1] was deleted; html[1] was inserted)
5. All in conjunction with Unbreakable

Reply:
1. I think it should. Something surely to test—will add one.
2. There is a test for no children. Will add one with children.
3. User responsibility (is `null`?)
4. We'll have to see about that in practice.
5. Well this simply doesn't work with the UnbreakableDriver. To have test be unbreakable, you need to give the complete context.